src/beamlime/applications/handlers.py

```diff
@@ -71,51 +53,43 @@ def process_message(self, message: MessageProtocol) -> MessageProtocol:
 def random_image_path() -> ImagePath:
     import uuid
 
-    return ImagePath(pathlib.Path(f"beamlime_plot_{uuid.uuid4().hex}.png"))
+    return ImagePath(pathlib.Path(f"beamlime_plot_{uuid.uuid4().hex}"))
```

Comment:
Can we change the name ``ImagePath`` to ``ImageNamePrefix``?
And together with ``--image-path`` argument parsing in ``executables/prototypes.py``, line #97.

Reply:
I think this may be removed soon anyway, if not, fix it when touching this code next.